api/restclient_test.go

```diff
@@ -0,0 +1,237 @@
+package api
```

Comment:
@suryagupta4 Please add the _// Copyright © 2024 Dell Inc. or its subsidiarie_s header